Zend/zend_alloc.c

```diff
@@ -69,6 +69,7 @@
 # include "win32/winutil.h"
 #endif
 
+#include <stdbool.h>
```

Comment:
This is probably unused?

Reply:
True. This commit was already merged (the rest of it), and this is what stgit left of it ... I need to drop it.